routes/web.php

```diff
@@ -122,6 +122,17 @@
             ]);
         }
 
+        if (!is_null($voucher->expires_at)) {
+            if ($voucher->expires_at < now()) {
```

Comment:
I noticed that in `ApiVoucherRedemptionsController`,  the comparison was done with `<=`. I'm sure the difference is negligible (only one second or millisecond?), but it seems worth being consistent. I think `<=` is more correct, so I suggest changing it here:
```suggestion
            if ($voucher->expires_at <= now()) {

```